tests/unit/test_client_logging.py

```diff
@@ -0,0 +1,103 @@
+import logging
+from unittest import mock
+
+from google.api_core.client_logging import (
+    setup_logging,
+    initialize_logging,
+)
+
+
+def reset_logger(scope):
+    logger = logging.getLogger(scope)
+    logger.handlers = []
+    logger.setLevel(logging.NOTSET)
+    logger.propagate = True
+
+
+def test_setup_logging_w_no_scopes():
+    with mock.patch("google.api_core.client_logging._BASE_LOGGER_NAME", "foo"):
```

Comment:
nit (not blocking) : For readability and a more informative mock, consider s/`foo`/`foogle`/g or s/`foo`/`test_base_logger`/g everywhere in this file. We can also do this as a follow-up.

(Again, not a blocker)